clusterDeployer.py

```diff
@@ -125,27 +128,61 @@ def remove_worker_dhcp_entries(self, hosts: List[ClusterHost]) -> None:
             h.prepare_remove_worker_dhcp_leases(self._local_host.bridge, removed_macs)
         lh = host.LocalHost()
         lh.run("systemctl restart libvirtd")
+        self._local_host.ensure_linked_to_network(self._local_host.bridge)
```

Comment:
Same question here.  I'm not sure I understand why removing dhcp entries would require re-adding the api port to the virtual bridge.

Reply:
Same as above - virsh net-destroy net-start has a side effect of removing the api port from virbr0. So, if we only remove workers and then add workers, we would end up with no api port on virbr0 on localhost.